yts/test_suite_test.go

```diff
@@ -91,8 +136,6 @@ func getEvents(in []byte) (string, error) {
 }
 
 func runTest(t *testing.T, testPath string) {
-	t.Helper()
```

Comment:
I didn't launch the tests, but the removal of this line is strange to me

This is a test helper, no?

Error would be reported were they fail.

But maybe it's the purpose of the changes.

Could you add t.Helper linter via golangci-lint and add a `//nolint:thelper // explanation `